cypress/e2e/integration/MainPage.cy.ts

```diff
@@ -0,0 +1,141 @@
+import { DateProps } from "interfaces/date_props.interface";
+
+describe("Renders Main page of app", () => {
+  let time: number = 0;
+  const lengthOfItems = 5;
+  const tasksRepeatableClass =
+    '[data-testid="task-list-box"] > [data-testid="task-list-items"]';
+
+  it("Renders correctly", () => {
+    cy.visit("/");
+  });
+
+  it("Testing date", () => {
+    const dateRepeatableClass =
+      '[data-testid="header-box"]> [data-testid="header-date-wrapper"]';
+    const today = new Date(Date.now());
+    const convertedDate: DateProps = {
+      day: today.getDate(),
+      month: today.toLocaleString("default", { month: "long" }),
+      year: today.getFullYear(),
+      currentDay: today.toLocaleString("default", { weekday: "long" }),
+    };
+    cy.get(
+      `${dateRepeatableClass} > [data-testid="header-day-wrapper"] >[data-testid="header-day"]`
+    ).should("have.text", convertedDate.day);
```

Comment:
If the text isn't related to the flow behavior, move the test to unit test in server env. instead.